msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/PermissionLevelManagerImpl.java

```diff
@@ -302,12 +302,12 @@ public DBMembershipItem createDBMembershipItem(String name, String permissionLev
 		return newDBMembershipItem;		
 	}
   
-  public void saveDBMembershipItem(DBMembershipItem item){
-  	getHibernateTemplate().saveOrUpdate(item);
+  public DBMembershipItem saveDBMembershipItem(DBMembershipItem item){
+		return getHibernateTemplate().merge(item);
```

Comment:
remove the use of HibernateTemplate here and in other places

Reply:
Done where affected!